File: scratch/read_old_footer.py
import subprocess

def get_old_styles():
    # Run git show to get styles.css content at 0ca5f96
    content = subprocess.check_output(["git", "show", "0ca5f96:styles.css"], text=True)
    lines = content.splitlines()
    
    # Let's search for footer lines
    footer_lines = []
    in_footer_section = False
    for i, line in enumerate(lines):
        if ".site-footer" in line or "footer-" in line:
            print(f"L{i+1}: {line}")

if __name__ == "__main__":
    get_old_styles()

File: scratch/read_old_footer.js
const { execSync } = require('child_process');

try {
  const content = execSync('git show 0ca5f96:styles.css', { encoding: 'utf8', maxBuffer: 10 * 1024 * 1024 });
  const lines = content.split('\n');
  lines.forEach((line, idx) => {
    if (line.includes('.site-footer') || line.includes('.footer-') || line.includes('footer-')) {
      console.log(`L${idx + 1}: ${line}`);
    }
  });
} catch (err) {
  console.error(err);
}
